src/routes/(auth)/login/+server.ts

```diff
@@ -81,6 +82,7 @@ export const POST = async ({
 
     return new Response(
       JSON.stringify({
+        error: 'invalid_credentials',
         message: 'Invalid Credentials'
       }),
```

Comment:
Given that we are throwing on L30, I don't think we could get here.

This error code is likely to be meet by catching on a `catch` clause.